apps/researcher/src/app/[locale]/revalidate/page.tsx

```diff
@@ -0,0 +1,21 @@
+import {redirect} from 'next/navigation';
+import {revalidatePath} from 'next/cache';
+
+interface Props {
+  searchParams?: {
+    path?: string;
+    redirect?: string;
+  };
+}
+
+export default async function RevalidatePaths({searchParams = {}}: Props) {
```

Comment:
To be sure: the interface definition of `Props` says `searchParams` can be undefined. But in the function signature `{}` is always set to `searchParams`. Is the interface definition correct then - shouldn't the `?` be omitted?

Reply:
I think it's best to make `searchParams` mandatory.